changelog_unreleased/blog-post-intro.md

```diff
@@ -0,0 +1,9 @@
+---
+author: "Sosuke Suzuki (@sosukesuzuki)"
+authorURL: "https://github.com/sosukesuzuki"
+title: "Prettier 3.4: Fixes a lot of bugs"
```

Comment:
```suggestion
title: "Prettier 3.4: A lot of bug fixes"
```

Optional: I could not find a blog post title starting with a verb, but there was https://prettier.io/blog/2019/06/06/1.18.0 for example